packages/pwa-kit-cli/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Build tools for pwa-kit",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mobify/mobify-platform-sdks.git"
+    "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
```

Comment:
Do you also want to add the `"homepage"` and `"bugs"` keys for the packages that are not private, as we recently did on V1?

https://github.com/SalesforceCommerceCloud/pwa-kit/pull/257/files

Reply:
Good call, I've added them in